btb/benchmark/__init__.py

```diff
@@ -11,6 +13,36 @@
 ]
 
 
+DEFAULT_MLCHALLENGES = [
+    CensusRF
+]
+
+
+def evaluate_candidate(name, candidate, challenges, iterations):
+    candidate_result = []
+
+    if not isinstance(challenges, list):
+        challenges = [challenges]
+
+    for challenge in challenges:
+
+        # Allow class and instance usage for challenges
+        if inspect.isclass(challenge):
```

Comment:
Maybe use [issubclass](https://docs.python.org/3/library/functions.html#issubclass) instead of `inspect`?